adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -144,7 +144,7 @@ Using the AWS Console, perform the following actions:
 * Create a folder within the bucket.
 * Upload the SAP media to the folder in the S3 bucket.
 
-image::s3_bucket.png[width=470]
+image::images/src/jpg/TRD_SLES-SAP-HA-automation-quickstart-cloud-s3-bucket.jpg[width=470]
```

Comment:
Same here - try it with just 

image::TRD_SLES-SAP-HA-automation-quickstart-cloud-s3-bucket.jpg[width=470]